config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -1737,7 +1737,7 @@
 // @section homing
 
 // The center of the bed is at (X=0, Y=0)
-//#define BED_CENTER_AT_0_0
+#define BED_CENTER_AT_150_150
```

Comment:
center of the 300x300 print surface

Reply:
this option is only for Delta and SCARA printers that require x0 y0 to be in the center.
changing this define the way you have literally does nothing as it links no where.

Reply:
Ok I will check this, maybe we should add a comment in that way then

Reply:
there are comments in place you just did not back trace the define to see them.

Reply:
I have looked for a comment like that but have not found. 
Can you indicate the line please ?

Reply:
Marlin/src/inc/Conditionals_post.h
Line 125

Reply:
I will change it back to original and will just add the comment from Conditionals_post.h by the way.

The comment is : 
// Require 0,0 bed center for Delta and SCARA

It don't explicitly tells that it's not needed at all in other cases. But with what I understand now, it's not needed to have it defined in Configuration.h

If it is already automatically defined in Conditionals_post.h , can we not just remove those 2 lines from Configuration.h ?

Reply:
It may be someone's preference to set the bed center to 0, 0 for a Cartesian, so the option is exposed.